scripts/ci/tags.yaml

```diff
@@ -477,3 +477,20 @@ cellular:
     - modules/lib/cjson/
     - modules/lib/hostap/
     - modules/lib/zcbor/
+
+edge_impulse:
+  files:
+    - modules/crypto/mbedtls/
```

Comment:
Edge Impulse by itself does not depend on many of these files (similarly to Event Manager).

Reply:
I did compilation of all samples and configuration from this sample and those crypto files are compiled.
However, list of decencies listed for tag can be shorter than list of compiled modules - I already did that by removing whole kernel from zephyr.
So this dependency exist but I can remove it if is not important, what do you think?

Reply:
Discussed offline. Dependencies related to boards were removed.